### PR TITLE
Move PCQ config into prod files

### PR DESCRIPTION
**Pull request by Cotterale**
+ Delete old pcq files

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by Cotterale** [comment]
Hold off on merging for now, this planning to be merged next

**Review by davejones74 — APPROVED** [review]

File: definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState.json [hunk 1]
```diff
@@ -2259,5 +2259,299 @@
     "CaseStateID": "DNisRefused",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/06/2020",
```
Comment (yarpo) [inline definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState.json:2264]: `LiveFrom` is ignored, but just to make it easier to understand when things were added I'd suggest to use date of release.

As I said - it's completely ignored by CCD.

File: test/unit/definitions/divorce/CaseEventToFields.test.js [hunk 2]
```diff
@@ -29,7 +29,6 @@ function mergeCaseFieldJsonNonProdFiles() {
     .concat(load('definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json'))
     .concat(load('definitions/divorce/json/CaseField/CaseField-general-order-nonprod.json'))
     .concat(load('definitions/divorce/json/CaseField/CaseField-general-email-nonprod.json'))
```
Comment (yarpo) [inline test/unit/definitions/divorce/CaseEventToFields.test.js:31]: please add `;` at the end of this line.

**Review by yarpo — APPROVED** [review]
If you want to test prod-like spreadsheet on `aat` or `demo`, please make sure you will manually generate spreadsheet.

File: `package.json`

is:
```
    "generate-excel-aat": "DIV_ENV=aat CCD_DEF_COS_URL=$npm_package_config_aat_cosUrl CCD_DEF_CCD_URL=$npm_package_config_aat_ccdUrl yarn run generate-excel -e *-prod.json",
```

update (temporarily):
```
    "generate-excel-aat": "DIV_ENV=aat CCD_DEF_COS_URL=$npm_package_config_aat_cosUrl CCD_DEF_CCD_URL=$npm_package_config_aat_ccdUrl yarn run generate-excel -e *-nonprod.json",
```
^ this will exclude `*-nonprod.json` files.

spreadsheet will be generated in folder:
`div-ccd-definitions/definitions/divorce/xlsx`

Upload it to `aat` (or `demo` if you generate a file for demo - `generate-excel-demo`) manually and do regression testing.

Before you release it, please make sure PET team is informed. (@Cotterale, @gurarach, @mshhmcts, Tom Seelig and Alison Coote).

Do not push your temporary updates from `package.json` file.

**Comment by yarpo** [comment]
I have converted it to draft to make sure it will not be accidentally merged.

When everything is ready for release please press button "Ready for release"

Screenshot 2020-09-11 at 16 40 50

File: definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState.json (hunk 1 above)
Comment (davejones74) [reply definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState.json]: Resolved using Mondays go live date for the SS 14/09/2020

**Pull request merged**